sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -690,7 +726,17 @@ public async Task GetPropertiesAsync()
             Response<BlobContainerProperties> response = await test.Container.GetPropertiesAsync();
 
             // Assert
-            Assert.AreEqual(PublicAccessType.BlobContainer, response.Value.PublicAccess);
+            Assert.IsNotNull(response.Value.LastModified);
+            Assert.IsNotNull(response.Value.LeaseStatus);
+            Assert.IsNotNull(response.Value.LeaseState);
+            Assert.IsNotNull(response.Value.LeaseDuration);
+            Assert.IsNotNull(response.Value.PublicAccess);
+            Assert.IsNotNull(response.Value.HasImmutabilityPolicy);
+            Assert.IsNotNull(response.Value.HasLegalHold);
+            Assert.IsNotNull(response.Value.DefaultEncryptionScope);
```

Comment:
This comes back not null for a default test container?  Will this need to be wrapped in a version check?

Reply:
fixed.